src/plugins/navigation-data/public/js/navigation-data.js

```diff
@@ -4,32 +4,6 @@
   plugins.navigationData = function (cockpit) {
     var self = this;
     self.cockpit = cockpit;
-    
-    this.actions = 
-    {
-      'plugin.navigationData.toggleHeadsUpDisplay':
-      {
-        description: "Toggle HUD",
-        controls:
-        {
-          button:
-          {
-            down: function() {
-              cockpit.emit('plugin.navigationData.toggleHeadsUpDisplay');
-            }           
-          }
-        }
-      }
-    };
-
-    this.inputDefaults = 
-    {
-      keyboard:
-      {
-        "\\": { type: "button",
-                action: 'plugin.navigationData.toggleHeadsUpDisplay' }
-      }
-    };
 
```

Comment:
I don't see the toggle HUD elements default being updated to something new. Was this duplicate code that was removed or is it missing?